google/async_resumable_media/_download.py

```diff
@@ -379,6 +359,9 @@ def _process_response(self, response):
         headers = self._get_headers(response)
         response_body = self._get_body(response)
 
+        responsebody1 = copy.copy(response_body)
```

Comment:
not pythonic name. please use snake case. also, perhaps a comment on why you are making  a copy of response_body?